src/app/queue/services/usecases.go

```diff
@@ -0,0 +1,31 @@
+package services
+
+import "clinic-api/src/app/queue"
+
+type usecase struct {
+	repo queue.Repositories
+}
+
+// CreateQueue implements queue.Services
+func (uc *usecase) CreateQueue(queue queue.Domain) (string, error) {
+	return uc.repo.InsertData(queue)
+}
+
+// GetAllQueues implements queue.Services
+func (uc *usecase) GetAllQueues(polyclinic, from string) ([]queue.Domain, error) {
```

Comment:
Prefer use `fromDate` or something else instead of `from` only. Because using single adjective words as a variable name may be ambiguous.

Reply:
https://github.com/ALTERA-CAPSTON-41/backend/pull/27/commits/c35f7d5577c7a15f6b71be7710564630f9306bd4
Done